dwh/validate_indicators.sh

```diff
@@ -49,51 +49,42 @@ echo "Output indicators file is: ${TEMP_OUT}"
 spark-submit indicators.py --src_dir=./test_files \
   --last_date=2020-12-30 --num_days=28 --output_csv=${TEMP_OUT}
 
-# PVLS Test
-counts=$(cat ${TEMP_OUT} | awk -F, '
-    BEGIN {num_true = 0; num_false = 0; num_none = 0;}
-    /False,ALL-AGES_ALL-GENDERS/ {num_false=$3}
-    /True,ALL-AGES_ALL-GENDERS/ {num_true=$3}
-    /None,ALL-AGES_ALL-GENDERS/ {num_none=$3}
-    /True,25-49_male/ {num_male_25=$3}
-    END {printf("%d,%d,%d,%d", num_true, num_false, num_none, num_male_25); }')
+##########################################
+# Assertion function that tests aggregates generated
+# by comparing them to what is expected
+# Arguments:
+#   $expected_aggr which is the expected aggregates
```

Comment:
You don't need to specify the name of the local variable here; if you want to be explicit, use `$1`, `$2`, ...